libbpf-rs/src/query.rs

```diff
@@ -0,0 +1,249 @@
+//! Query the host about BPF
```

Comment:
this file is not supposed to be in this PR, right?

Reply:
It's b/c this PR is stacked on top of the query PR which hasn't landed yet. Once the lower PRs land it'll just be the example code. I left it in draft mode b/c of that